LPS-199417 | Testing

allen-nagtalon opened the pull request:
https://liferay.atlassian.net/browse/LPS-199417

CalumR23 committed:
POSHI-577 Update unit tests with new syntax

CalumR23 committed:
POSHI-577 Test fix to pass in proper var

CalumR23 committed:
POSHI-577 Test fix

achaparro committed:
LPS-196393 Use random companyIds to avoid collisions in LXC

CalumR23 committed:
POSHI-577 Open syntax change

CalumR23 committed:
POSHI-577 Pause syntax change

achaparro committed:
LPS-196393 Tests: add a check anytime we create a company to be sure it is random and we do not use the counter

nNilton committed:
LPS-197343 Create new bundle tracker for Site Initializer Client Extension

nNilton committed:
LPS-197343 Detect Site Initializer Client Extension and convert to OSGI Bundle in WabProcessor

AliciaGarciaGarcia committed:
LPS-197819 create the new fragment and only show it if the FF is active

balazssk committed:
LPS-198081 Draft Layouts are handled by their approved Layout

smottal committed:
COMMERCE-12430 Fix API

smottal committed:
COMMERCE-12430 Update autocomplete component

smottal committed:
COMMERCE-12430 Search in organization chart

smottal committed:
COMMERCE-12430 Update package dependencies

nNilton committed:
LPS-197343 prevent NPE

nNilton committed:
LPS-197343 _isAlreadyProcessed

CalumR23 committed:
POSHI-577 Change param name to functionType

balazssk committed:
LPS-198082 Add isOriginalPrivateLayout to store which LayoutSet Layouts are imported into

linolaoj committed:
LPS-189931 checking if indexer exists

smottal committed:
COMMERCE-12430 Fix tests

Commit:
LPS-198548 Check account restricted into organization scope as well for objectDefinition2, in oneToMany and manyToMany cases. Remove unused methods and variable

CalumR23 committed:
POSHI-577 Add new unit tests for Poshi signature change

CalumR23 committed:
POSHI-577 Add methods to call new unit tests

balazssk committed:
LPS-198082 LayoutStagedModelDataHandler will handle this

Commit:
LPS-198548 Integration test: adjust testGetObjectEntryRelatedObjectEntriesWithAccountEntryRestricted

CalumR23 committed:
POSHI-577 SelectPopUp changes

nNilton committed:
LPS-197343 simplify

Commit:
LPS-198548 Integration test: add organization scope scenario into testGetObjectEntryRelatedObjectEntriesWithAccountEntryRestricted

nNilton committed:
LPS-197343 fix site api to correct render all types of fragment

nNilton committed:
LPS-197343 typo

jonathanmccann committed:
LPS-198970 Ensure the correct company ID in order to reindex properly when the database is partitioned

CalumR23 committed:
POSHI-577 Change to use JS function

CalumR23 committed:
POSHI-577 SelectWindow changes

smottal committed:
COMMERCE-12430 Fix search

AliciaGarciaGarcia committed:
LPS-197819 FF

CalumR23 committed:
POSHI-577 AssertElement changes

pietrobocale committed:
COMMERCE-12430 Add new paths

balazssk committed:
LPS-198082 Extend private importing pattern to LayoutPrototypes

CalumR23 committed:
POSHI-577 Pass in hard coded arguement

CalumR23 committed:
POSHI-577 Do not pass in value for waitElementPresent

dsitu committed:
LPS-195988 Increase AccountEntry name character limit to 250

dsitu committed:
LPS-195988 Regen

sergiojimcos committed:
LPS-191346 Create PaginationProvider to use this code in REST API and GraphQL and avoid duplication

sergiojimcos committed:
LPS-191346 Apply PaginationProvider to PaginationContextProvider for REST API

CalumR23 committed:
POSHI-577 WaitForElementPresent and WaitForVisible changes

sergiojimcos committed:
LPS-191346 Apply PaginationProvider for GraphQL

Commit:
LPS-197343 some cleanup

Commit:
LPS-199457 Delete UpgradeJavaCaptchaUtilCheck

Commit:
LPS-199457 Add new test cases for UpgradeCatchAllCheck

CalumR23 committed:
POSHI-577 Change to hard coded paths

Commit:
LRQA-83365 Improve task on test

Commit:
LPS-163167 Added method for the retrieval of an operation schema

Commit:
LPS-163167 Usage of new method + prevent NPE by checking if the operation schema is empty

Commit:
LPS-163167 Added autogeneration of headers in case no schema/requestbody is specified

Commit:
LPS-163167 buildREST

Commit:
LPS-163167 Removal of unnecesary schema in endpoint response

Commit:
LPS-198548 Integration test: add manyToMany scenario into testGetObjectEntryRelatedObjectEntriesWithAccountEntryRestricted

sergiojimcos committed:
LPS-191346 Add some test cases

CharlotteFrancis committed:
LRCI-3810 move generate-testray-csv logic into jenkins results parser

CharlotteFrancis committed:
LRCI-3810 SF

CalumR23 committed:
POSHI-577 Change selectPopUp to value1

CalumR23 committed:
POSHI-577 Pass in null value into confirm

CalumR23 committed:
POSHI-577 openNoErrorNoSPARefresh changes

CalumR23 committed:
POSHI-577 Hard code value into function call

sergiojimcos committed:
LPS-191346 Create PaginationConfigurationTestUtil to avoid duplicate code

sergiojimcos committed:
LPS-191346 Apply util changes

22LucaGiannotta22 committed:
LPS-195988 Adapt functional test

Commit:
LPS-163167 Revert of "LPS-163167 Added autogeneration of headers in case no schema/requestbody is specified"

Commit:
LPS-163167 Modification of the existing rest-openapi yamls for adapting to the new behavior

BarbaraCabrera committed:
LPS-199729 Reverting code "LPS-198515 Only handle the error component instead of the whole event object" and SF renaming

Commit:
LPS-199503 batch-planner-rest-impl - add logic that determines scope of the object

Commit:
LPS-199503 object-rest-impl - not sure why we take filter from the request instead of method param, but batch engine doesn't work with httpServletRequests so we need this ugly condition

Commit:
LPS-199503 batch-engine-api - for site scoped entities, object resources expect scopeKey parameter, other resources have siteId parameter

dsitu committed:
LPS-195988 Fix search container css

Commit:
LPS-197098 Hardcode

noornajj committed:
LPS-199053 Add move URL to ctEntry view

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

allen-nagtalon commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>default</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/3922/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3922/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:default - 0 out of 2 jobs passed in 2 hours 18 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/96">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1257401_3922</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a58682d046eb2821b21d550c683c4351863af365">a58682d046eb2821b21d550c683c4351863af365</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6142a2fff64a4669c36a802ab1701404be04a796">6142a2fff64a4669c36a802ab1701404be04a796</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2444748625">EE Development Acceptance (master) - 95 - 2023-10-20[08:43:21]</a></p><details><summary><strong>ci:test:default - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/3922/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/773201/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3922/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/773201//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/773201//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ManagePublications_AssertDependenciesMovePublicationsWithParentChange_">LocalFile.ManagePublications#AssertDependenciesMovePublicationsWithParentChange</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_AssertDependenciesMovePublicationsWithParentChange/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_AssertDependenciesMovePublicationsWithParentChange/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_AssertDependenciesMovePublicationsWithParentChange/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/773201//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ManagePublications_CanMoveChangeBetweenPublications_">LocalFile.ManagePublications#CanMoveChangeBetweenPublications</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_CanMoveChangeBetweenPublications/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_CanMoveChangeBetweenPublications/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_CanMoveChangeBetweenPublications/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/773201//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ManagePublications_CannotMoveChangesBetweenPublicationsIfConflictsArePresent_">LocalFile.ManagePublications#CannotMoveChangesBetweenPublicationsIfConflictsArePresent</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_CannotMoveChangesBetweenPublicationsIfConflictsArePresent/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_CannotMoveChangesBetweenPublicationsIfConflictsArePresent/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ManagePublications_CannotMoveChangesBetweenPublicationsIfConflictsArePresent/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-10/test-1-25/test-portal-acceptance-pullrequest(master)/3922/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/3922/">test-portal-acceptance-pullrequest(master)#3922</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3922/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>default<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/allen-nagtalon/liferay-portal/pull/33">allen-nagtalon#33</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2444762212">[master] ci:test:default - allen-nagtalon &gt; allen-nagtalon - PR#33 - 2023-10-20[17:39:25]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-25.liferay.com/job/publish-testray-report/7854/">publish-testray-report#7854</a><br/></div></div>

Commit:
LPS-198508 batch-planner-test - add test for export of site scoped object entries

pietrobocale committed:
COMMERCE-12430 Add new macro addAccountEntrySubOrganization

pietrobocale committed:
COMMERCE-12430 Add new JSON macro _deleteOrganizationById

pietrobocale committed:
COMMERCE-12430 Add new tests

pietrobocale committed:
COMMERCE-12430 Auto SF

JanisZhang committed:
LPS-199703 Replace non-blocking ServiceProxyFactory usages by using Snapshot in ClientDataRequestHelperUtil

JanisZhang committed:
LPS-199703 Replace non-blocking ServiceProxyFactory usages by using Snapshot in DefaultActionableDynamicQuery

JanisZhang committed:
LPS-199703 Replace non-blocking ServiceProxyFactory usages by using Snapshot in JSONWebServiceActionsManagerUtil

sergiojimcos committed:
LPS-191346 Combine if-else correctly

sergiojimcos committed:
LPS-191346 Simplify

alberttgomes committed:
LPS-199343 Remove the UpgradeJavaCommerceCountryServiceCheck

alberttgomes committed:
LPS-197142 Update imports.txt

sandrodw3 committed:
LPS-199789 Move keyboard movement components to new folder

alberttgomes committed:
LPS-197142 Update replacements.json

alberttgomes committed:
LPS-197142 Add new test cases for UpgradeCatchAllCheck

nNilton committed:
ISSD-1407 fix zip path

dsitu committed:
LPS-195988 Fix commerce affected pages for long account names

CalumR23 committed:
POSHI-577 waitForSelectPopUp changes

CalumR23 committed:
POSHI-577 Hard code URL instead of using path

CalumR23 committed:
POSHI-577 Follow proper poshi parameter rules

JanisZhang committed:
LPS-199841 Replace non-blocking ServiceProxyFactory usages by using Snapshot in SocialActivityManagerUtil

JanisZhang committed:
LPS-199841 Replace non-blocking ServiceProxyFactory usages by using Snapshot in MicrosoftTranslatorFactoryUtil

JanisZhang committed:
LPS-199841 Replace non-blocking ServiceProxyFactory usages by using Snapshot in PortletDisplayTemplateManagerUtil

JanisZhang committed:
LPS-199841 Replace non-blocking ServiceProxyFactory usages by using Snapshot in TemplateHandlerRegistryUtil

izaera committed:
LPS-198883 Unmount React components properly

izaera committed:
LPS-198883 Give React component a proper name

Otherwise it doesn't show up in React dev tools plugin because it is
anonymous.

ccorreagg committed:
LPS-199503 Use Util class

ccorreagg committed:
LPS-199503 Add method to get an Object Definition based on its name and company id. The fetch method already exists and we need the one that throws an exception if it does not exist

ccorreagg committed:
LPS-199503 buildService

ccorreagg committed:
LPS-199503 Use get method to throw the proper exception if the object definition does not exist

ccorreagg committed:
LPS-199503 Inline

sandrodw3 committed:
LPS-199789 Restore keyboard movement preview CSS

ccorreagg committed:
LPS-199503 Unify

ccorreagg committed:
LPS-199503 Move support for the siteId/scopeKey parameter to the Object Entry VulcanBatchEngineTaskItemDelegate implementation. Batch does not have to understand what scopeKey is

Stephy6 committed:
ISSD-1406 Remove unused postMessage because there isn't any initMessage value passed to the init function

Stephy6 committed:
ISSD-1406 Delete old main.js AUI file, its usage and files

Stephy6 committed:
ISSD-1406 Create new MarketplaceFrame.js as a substitute of main.js AUI file

Stephy6 committed:
ISSD-1406 Usage of MarketplaceFrame.js component with liferay-frontend:component taglib

gergelyszalay committed:
LPS-199877 Change to get local date from calendar with CalendarUtil

DiegoHu97 committed:
LPS-199861 Show parent title when site has parent site

claraizquierdo committed:
LPS-179670 Do not allow to drag a disabled element and set enabled again when the action has finished

ccorreagg committed:
LPS-199503 Add test

allen-nagtalon commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>default</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11661/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11661/jenkins-report.html">here</a>.</b></p>

CalumR23 committed:
POSHI-577 Remove duplicate function

CalumR23 committed:
POSHI-577 Change js parameters

pedroliveirav committed:
ISSD-1441 Fix delay when change complimentary key permission

david-truong committed:
LPS-198982 add CT display renderer for LayoutPageTemplateEntry

david-truong committed:
LPS-198982 don't hide draft layout if it is in draft state

david-truong committed:
LPS-198982 SF

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:default - 2 out of 2 jobs passed in 2 hours 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/700a78e8165928d580f3406d40904d369ce060d7">700a78e8165928d580f3406d40904d369ce060d7</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/02a1313f6cf211c198351eeafcacfc05a462da48">02a1313f6cf211c198351eeafcacfc05a462da48</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2446236140">EE Development Acceptance (master) - 106 - 2023-10-24[04:44:32]</a></p><details><summary><strong>ci:test:default - 2 out of 2 jobs PASSED</strong></summary><details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11661/">test-portal-acceptance-pullrequest(master)</a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/755825/">functional-tomcat90-mysql57-jdk8/0/0</a></li></ul></li></ul></details></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11661/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11661/">test-portal-acceptance-pullrequest(master)#11661</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11661/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>default<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/allen-nagtalon/liferay-portal/pull/33">allen-nagtalon#33</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2446375298">[master] ci:test:default - allen-nagtalon &gt; allen-nagtalon - PR#33 - 2023-10-24[10:53:31]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/26929/">publish-testray-report#26929</a><br/></div></div>

Commit:
ISSD-1430 Add accountId to the route path

Commit:
ISSD-1430 Create logic to display or not add member button

pei-jung committed:
LPS-198982 Prevent publication with draft changes from being published

JerryAdrianoo committed:
LPS-190313 Add CanGoToViewPage test

JerryAdrianoo committed:
LPS-190313 Add CanGoToEditViewPage test

JerryAdrianoo committed:
LPS-190313 Add CanViewRenameOption test

pei-jung committed:
LPS-198982 Set layout status to approved when LayoutPageTemplateEntry is published

pei-jung committed:
LPS-198982 Update LayoutPageTemplateEntry edit url to include tabs1 param

GergoMathe committed:
LPS-196410 Adding new path element

GergoMathe committed:
LPS-196410 Adding new test cases

L

AliciaGarciaGarcia committed:
LPS-197819 buildLang

victorg1991 committed:
LPS-184267 Make sure to use the label since the value may be different ant we use the label to filter

4lejandrito committed:
LPS-189508 Adding failing test

markocikos committed:
LPS-199992 Fix property cleanup. It was not removed if it is set to `false`.

pietrobocale committed:
COMMERCE-12430 Improve tests

JerryAdrianoo committed:
LPS-190313 Adding fix to the CX testcase file

ccorreagg committed:
LPS-199503 Fix test. The groupId must be 0 if the object definition is scoped by company, otherwise an exception is thrown because of a validation

4lejandrito committed:
LPS-189508 Adding support for custom url template params

4lejandrito committed:
LPS-189508 Removing deprecated call

4lejandrito committed:
LPS-189508 Resolving erc and action name in the action href

magjed4289 committed:
LPS-195509 CanSeeGetEndpointPathWithIdParameterInAPIExplorer

ccorreagg committed:
LPS-199503 Add support when the internalClassNameKey is not valid (the VulcanBatchEngineTaskItemDelegate or the Object Definition do not exist)

Commit:
ISSD-1430 Fix accounts logic to display in Publisher Dashboard

Commit:
ISSD-1430 SF

ccorreagg committed:
LPS-189508 Set parameter only if the parameter is equal to 'id' or ends with 'Id'. With that if clause, we avoid setting parameters that are not related with the identifier, such as externalReferenceCodes, etc.

andreluis2022 committed:
LPS-195539 Adding new paths for Dat Set

CalumR23 committed:
POSHI-577 Change values to null in Confirm

allen-nagtalon commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>default</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/8060/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/jenkins-report.html">here</a>.</b></p>

allen-nagtalon commented:
ci:stop

liferay-continuous-integration commented:
<html><h3>:x: ci:test:default - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1206124b6f7b7d569ceaf75a45f8dffac6ab062f">1206124b6f7b7d569ceaf75a45f8dffac6ab062f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1206124b6f7b7d569ceaf75a45f8dffac6ab062f">1206124b6f7b7d569ceaf75a45f8dffac6ab062f</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2446650657">EE Development Acceptance (master) - 109 - 2023-10-24[16:55:01]</a></p><details><summary><strong>ci:test:default - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/8060/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/8060/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>Build was aborted</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

allen-nagtalon commented:
ci:test

CharlotteFrancis committed:
LRCI-3810 fix compile errors

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/8060/">test-portal-acceptance-pullrequest(master)#8060</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8060/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>default<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/allen-nagtalon/liferay-portal/pull/33">allen-nagtalon#33</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2446683289">[master] ci:test:default - allen-nagtalon &gt; allen-nagtalon - PR#33 - 2023-10-25[11:40:38]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-39.liferay.com/job/publish-testray-report/16506/">publish-testray-report#16506</a><br/></div></div>

liferay-continuous-integration commented:
<h1>All tests completed.</h1><p>Jenkins is currently processing test results.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1206124b6f7b7d569ceaf75a45f8dffac6ab062f">1206124b6f7b7d569ceaf75a45f8dffac6ab062f</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/8061/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8061/jenkins-report.html">here</a>.</b></p>

Commit:
LRAC-14775 Ignore tests related to the Acquisitions card

MicaelleSilvaa committed:
LPS-199634 Update replacements.json

MicaelleSilvaa committed:
LPS-199634 Update UpgradeCatchAllCheck.testjava with new testcases

MicaelleSilvaa committed:
LPS-199634 Delete UpgradeJavaLayoutServicesCheck

CalumR23 committed:
POSHI-577 Function signature update

Commit:
LRAC-14776 Ignore tests related to the Cohort Analysis card

CalumR23 committed:
POSHI-577 Macro signature syntax update

Commit:
LPS-195385 Fix SolrSearchFacets#ViewMultipleFacetURLs

Commit:
LRAC-14778 Update product attributes that appear on the first and last page

xbrianlee committed:
LPS-199807 Update steps for changing display template (additional commits)

allen-nagtalon commented:
ci:test

allen-nagtalon commented:
ci:stop

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>default</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/4758/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4758/jenkins-report.html">here</a>.</b></p>

dsitu committed:
COMMERCE-12551 Fix integration test

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:default - 2 out of 2 jobs passed in 1 hour 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1206124b6f7b7d569ceaf75a45f8dffac6ab062f">1206124b6f7b7d569ceaf75a45f8dffac6ab062f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1206124b6f7b7d569ceaf75a45f8dffac6ab062f">1206124b6f7b7d569ceaf75a45f8dffac6ab062f</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2446650657">EE Development Acceptance (master) - 109 - 2023-10-24[16:55:01]</a></p><details><summary><strong>ci:test:default - 2 out of 2 jobs PASSED</strong></summary><details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/4758/">test-portal-acceptance-pullrequest(master)</a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/285177/">functional-tomcat90-mysql57-jdk8/0/0</a></li></ul></li></ul></details></details><h5>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4758/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/4758/">test-portal-acceptance-pullrequest(master)#4758</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4758/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>default<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/allen-nagtalon/liferay-portal/pull/33">allen-nagtalon#33</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2446751049">[master] ci:test:default - allen-nagtalon &gt; allen-nagtalon - PR#33 - 2023-10-25[16:44:01]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-35.liferay.com/job/publish-testray-report/10382/">publish-testray-report#10382</a><br/></div></div>

ccorreagg committed:
LPS-189508 Move the filling of the map before validating if firstParameterName is null to do even if the parameter is null

ccorreagg committed:
LPS-189508 Split in two to have the if sorted

Mikellorza committed:
LPS-199579 Exclude guest role

Commit:
LPS-163167 Changing the order of a condition to make it more readable

ccorreagg committed:
LPS-199503 SF

dacousalr committed:
LPS-200117 Filter by companyId instead of groupId since there might be Web Contents created with the same structure in other child sites of the same virtual instance

lorenzolapescara committed:
COMMERCE-12673 Change Confirm to AssertConfirm

Commit:
LPS-197098 Remove redundant steps

shuyangzhou committed:
LPS-199563 Branch out for 7.4 to keep older branches unchanged.

sandrodw3 committed:
LPS-200004 Add missing open class

nicolas-sss committed:
LPS-199343 SF - Follow the pattern for testcases on UpgradeCatchAllCheck.testjava

nicolas-sss committed:
LPS-199343 Generated

brianchandotcom committed:
LPS-199563 prep next

brianchandotcom committed:
LPS-199563 prep next

MicaelleSilvaa committed:
LPS-199634 Generated

pyoo47 committed:
LRCI-3810 Rename

pyoo47 committed:
LRCI-3810 Make methods static

Commit:
LPS-197613 Update replacements.json

Commit:
LPS-197613 Add new test cases for UpgradeCatchAllCheck

brianchandotcom committed:
LPS-191346 SF

pyoo47 committed:
LRCI-3810 Only generate() should be public

MicaelleSilvaa committed:
LPS-199606 Update replacements.json

MicaelleSilvaa committed:
LPS-199606 Add new test cases for UpgradeCatchAllCheck

MicaelleSilvaa committed:
LPS-199606 Create the UpgradeCatchAllCheck.testjspf

CalumR23 committed:
POSHI-657 Update default poshi runner version

CalumR23 committed:
POSHI-657 Use latest gradle-plugins-poshi-runner

CalumR23 committed:
POSHI-657 Use latest gradle-plugins-workspace

CalumR23 committed:
POSHI-657 Update poshi-core version

CalumR23 committed:
POSHI-657 Prepare source-formatter

andrea-ale-sbarra committed:
COMMERCE-12676 fix availability checks

michaelhashimoto committed:
LRCI-3845 Shift to using build updater to determine whether to use a reinvoke rule

michaelhashimoto committed:
LRCI-3845 Catch a potential null pointer

Commit:
LPS-199457 Update replacements.json

Commit:
LPS-199457 Generated

nicolas-sss committed:
LPS-199606 Delete UpgradeGetClassNamesMethodCheck

nicolas-sss committed:
LPS-199606 Generated

CalumR23 committed:
POSHI-657 SF

pei-jung committed:
LPS-198982 Blank Site Initializer - set the status of the utility page layouts to approved

brianchandotcom committed:
LPS-197343 SF

CalumR23 committed:
POSHI-657 Generate CHANGELOG

brianchandotcom committed:
COMMERCE-12676 SF

brianchandotcom committed:
commerce-frontend-js 4.0.100 prep next

brianchandotcom committed:
commerce-frontend-taglib 22.0.2 prep next

brianchandotcom committed:
LPS-197343 Add warning for LPS-169057

brianchandotcom committed:
LPS-199165 temporarily use this hash

brianchandotcom committed:
POSHI-557 prep next

brianchandotcom committed:
POSHI-557 prep next

brianchandotcom committed:
POSHI-557 regen

brianchandotcom committed:
POSHI-557 prep next

brianchandotcom committed:
POSHI-557 prep next

brianchandotcom committed:
POSHI-557 prep next

andreluis2022 committed:
LPS-187618 Adding a new paths to DataSet

andreluis2022 committed:
LPS-187618 Adding some automation tests in DataSet

nicolas-sss committed:
LPS-197613 SF

brianchandotcom committed:
POSHI-557 auto SF

Commit:
LPS-197931 Update replacements.json

Commit:
LPS-197931 Add new test cases for UpgradeCatchAllCheck

michaelhashimoto committed:
LRCI-3845 Avoid updating the JenkinsCohort unless needed

michaelhashimoto committed:
LRCI-3845 Reset at the end of a build lifecycle

allen-nagtalon committed:
LPS-197401 Refactor CanMoveChangeBetweenPublications test and add task blocks

allen-nagtalon committed:
LPS-197403 Refactor CannotMoveChangesBetweenPublicationsIfConflictsArePresent test

brianchandotcom committed:
LRCI-3845 prep next

brianchandotcom committed:
LPS-196393 Sort

allen-nagtalon committed:
LPS-199417 Add test to assert dependencies move publications with their parent change

allen-nagtalon committed:
LPS-197401 Refactor macro to match new move changes UI

Commit:
LPS-199998 Add locator for dropdown subheader by index

Commit:
LPS-199998 Add locator for grouped dropdown menu item by index

Commit:
LPS-199998 Add case to view default display template order

Commit:
LPS-199998 Hardcode

Commit:
LPS-199998 Modify macro to make it reusable

Commit:
LPS-199998 Modify macro to support groupId

Commit:
LPS-199998 Add case to view new display templates order

Commit:
LPS-199998 Add case to select display template via keyboard

Commit:
LPS-199998 Add missing property

andrea-ale-sbarra committed:
COMMERCE-12676 removed outdated test

brianchandotcom committed:
LPS-200004 Sort

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-199503 SF

brianchandotcom committed:
LPS-163167 Only used in one place

brianchandotcom committed:
LPS-163167 prep next

brianchandotcom committed:
LPS-163167 prep next

brianchandotcom committed:
LRCI-3810 Rename

brianchandotcom committed:
LRCI-3810 SF

brianchandotcom committed:
LRCI-3810 prep next

brianchandotcom committed:
LPS-199457 prep next

brianchandotcom committed:
LPS-199457 prep next

allen-nagtalon commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-8.liferay.com/job/test-portal-source-format/6321/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-source-format/builds/6321/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 8 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/259e93090188cbf1a0ddca3d8d63a4b2d116212e">259e93090188cbf1a0ddca3d8d63a4b2d116212e</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/allen-nagtalon/liferay-portal/tree/LPS-197401-Testing">LPS-197401-Testing</a><br/>Branch GIT ID: <a href="https://github.com/allen-nagtalon/liferay-portal/commit/e2df6d538ad548c270781d8473b3ce2f3d1c961a">e2df6d538ad548c270781d8473b3ce2f3d1c961a</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-source-format/6321/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-source-format/builds/6321/jenkins-report.html">here</a>.</h5><pre>     [java] java.lang.Exception: Found formatting issues:
     [java] At least one commit message is missing a reference to a required JIRA project: BLADE, CLDSVCS, COMMERCE, FACES, IDE, ISSD, ISSITES, LCD, LHC, LOOP, LPS, LRAC, LRCI, LRDOCS, LRIS, LRQA, OAUTH2, POSHI, RELEASE, SYNC, TR. Please verify that the JIRA project keys are specifiedin source-formatter.properties in the liferay-portal repository.
     [java]   at com.liferay.source.formatter.util.JIRAUtil.validateJIRAProjectNames(JIRAUtil.java:44)
     [java]   at com.liferay.source.formatter.SourceFormatter._validateCommitMessages(SourceFormatter.java:1296)
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:327)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:291)
[stopwatch] [run.batch.test.action: 5:03.336 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:456: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:648: Java returned: 1
      [get] Getting: http://test-1-8/job/test-portal-source-format/6321//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20231027095833933.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20231027095833933.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1230200947.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[beanshell] Truncating errors in /opt/dev/projects/github/liferay-portal/portal-impl/test-results/TEST-JenkinsLogAssertTest.xml
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1846269636
[junitreport] Loading stylesheet jar:file:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 224ms
[junitreport] Deleting: /tmp/null1846269636
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [echo] run.batch.test.tear.down.start.timestamp: 10-27-2023 09:58:38:307 PDT
[stopwatch] [run.batch.test.tear.down: 0.000 sec]</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-8.liferay.com/job/test-portal-source-format/6321/">test-portal-source-format#6321</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-source-format/builds/6321/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/allen-nagtalon/liferay-portal/pull/33">allen-nagtalon#33</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2448028206">[master] ci:test:sf - allen-nagtalon &gt; allen-nagtalon - PR#33 - 2023-10-27[09:51:05]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-8.liferay.com/job/publish-testray-report/3996/">publish-testray-report#3996</a><br/></div></div>
